homeassistant/components/wyoming/config_flow.py

```diff
@@ -0,0 +1,53 @@
+"""Config flow for Wyoming integration."""
+from __future__ import annotations
+
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_PORT
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import DOMAIN
+from .info import load_wyoming_info
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST): str,
+        vol.Required(CONF_PORT): int,
+    }
+)
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Wyoming integration."""
+
+    VERSION = 1
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Handle the initial step."""
+        if user_input is None:
+            return self.async_show_form(
+                step_id="user", data_schema=STEP_USER_DATA_SCHEMA
+            )
+
+        wyoming_info = await load_wyoming_info(
+            user_input[CONF_HOST],
+            user_input[CONF_PORT],
+        )
+
+        if wyoming_info is None:
+            return self.async_abort(reason="cannot_connect")
+
+        # ASR = automated speech recognition (STT)
+        asr_installed = [asr for asr in wyoming_info.asr if asr.installed]
+        if not asr_installed:
+            return self.async_abort(reason="no_services")
+
+        name = asr_installed[0].name
+        data = {"asr": asr_installed[0].to_dict(), **user_input}
+
+        return self.async_create_entry(title=f"Wyoming ({name})", data=data)
```

Comment:
You don't need to add Wyoming to the title, as it will always be presented next to the name of the integration
```suggestion
        return self.async_create_entry(title=name, data=data)
```